doc/source/Resources/pyaedt_with_IDE.bat

```diff
@@ -33,7 +34,7 @@ if NOT [%usewheel%]==[] (
 	set wheelpyaedt="!argVec[%usewheel%]!"
 	if [%usepython%]==[] (
 	    echo ----------------------------------------------------------------------
-	    echo WheelHouse has been spefified. Make sure you are using version 3_7
+	    echo WheelHouse has been specified. Make sure you are using version 3_7
```

Comment:
Is there a reason that the format is 3_7 and not 3.7?

Reply:
@PipKat  this message was present before. I'll tag @maxcapodi78  as he checked in the original message